### DISCUSSION TITLE
just errors in the cmd, but not app breaking

### DISCUSSION THREAD

**Discussion by Holyniwa** [vladmandic/sdnext#140] [Show and tell]
I noticed these in the cmd window. they weren't there early this morning, but I cant say when they started. maybe when the Gradio thing happened? Since it wasn't breaking anything, I decided to just drop em here in case they hadn't been spotted by anyone else (or brought to attention). 

17:59:29-117453 ERROR    Error running git with args: -C "D:\stable-diffusion\automatic\extensions-builtin\LDSR" pull
                         --rebase --autostash
17:59:29-772651 ERROR    Error running git with args: -C "D:\stable-diffusion\automatic\extensions-builtin\Lora" pull
                         --rebase --autostash
17:59:30-461566 ERROR    Error running git with args: -C
                         "D:\stable-diffusion\automatic\extensions-builtin\prompt-bracket-checker" pull --rebase
                         --autostash
17:59:31-152101 ERROR    Error running git with args: -C "D:\stable-diffusion\automatic\extensions-builtin\ScuNET" pull
                         --rebase --autostash
17:59:34-771786 ERROR    Error running git with args: -C "D:\stable-diffusion\automatic\extensions-builtin\SwinIR" pull
                         --rebase --autostash

**Comment by vladmandic** [comment]
if you see errors like that, you should check `setup.log` for more details - maybe there's something there that explains it.
and it quite possibly could be github rate limiting so it will go away on its own after a while.
ideally, you should only do extension updates when main repo is updated (otherwise you can use `--quick` to skip all that), but since i have a brand new installer, i'm still tweaking defaults.

**Reply by Holyniwa** [reply]
i was going to ask if there's a way to control when to update or not (considering what happened with a1111) but figured you may need the feedback since this is still a WIP
